### PR TITLE
#Add Bottom Navigation Bar on Allergen Alert Screen

### PR DESCRIPTION
**Pull request by AntonisFaros**
## Description
Removed the floating action button and added a bottom navigation bar on Allergen Alert Screen

## Related issues and discussion
#2397 
 
 ## Screen-shots
Screenshot

### CODE REVIEW

File: app/src/main/res/layout/fragment_alert_allergens.xml [hunk 1]
```diff
@@ -1,57 +1,117 @@
...
+        android:layout_gravity="top|center"
+        android:layout_marginHorizontal="10dp"
+        android:clickable="true"
+        android:drawableStart="@drawable/plus_blue"
+        android:paddingStart="20dp"
+        android:paddingEnd="20dp"
+        android:drawablePadding="-20dp"
+        android:focusable="true"
+        android:layout_marginTop="10dp"
+        android:text="Add an allergen"
```
Comment (teolemon) [inline app/src/main/res/layout/fragment_alert_allergens.xml]: Can you extract the string ?

Comment (teolemon) [reply app/src/main/res/layout/fragment_alert_allergens.xml]: To be honest, I'd like to apply this to as many screens as possible.

Comment (AntonisFaros) [reply app/src/main/res/layout/fragment_alert_allergens.xml]: @teolemon I am not sure how to do that , but i can try my best.

Comment (teolemon) [reply app/src/main/res/layout/fragment_alert_allergens.xml]: @deniger @AntonisFaros let's merge as it, then

**Pull request merged**